src/js/services/conversation-service.js

```diff
@@ -163,3 +163,10 @@ export function handleConversationUpdated() {
 
     return Promise.resolve();
 }
+
+export function postPostback(actionId) {
+    const {user} = store.getState();
+    return core().conversations.postPostback(user._id, actionId).then((response) => {
```

Comment:
because `postPostback` already resolve with the response, you're adding a useless layer